README.md

```diff
@@ -71,6 +71,18 @@ Configure default setting for MemGPT by running:
 ```
 memgpt configure
 ```
+Configurations to  be done:

```

Comment:
Can you remove this diff from the PR? Seems unrelated

Reply:
Yes sure